test/sse/test_sse4_1.cpp

```diff
@@ -19,94 +19,177 @@ int numInterestingInts = sizeof(interesting_ints_)/sizeof(interesting_ints_[0]);
 double *interesting_doubles = get_interesting_doubles();
 int numInterestingDoubles = sizeof(interesting_doubles_)/sizeof(interesting_doubles_[0]);
```

Comment:
I wonder if there is some `#pragma` to turn on inlining for the whole file?  Maybe not?

Reply:
The only other possibility I suppose would be to build with `-fno-inline-functions` though that might affect more things.